inference-engine/tests/functional/inference_engine/CMakeLists.txt

```diff
@@ -216,3 +216,7 @@ else()
     #                                          FLAGS  "/we4996 /W4 /WX"
     #                                          PLUGIN_API)
 endif()
+
+
+# ir serialization functional tests variables
+add_definitions("-DIR_SERIALIZATION_MODELS_PATH=\"${CMAKE_CURRENT_SOURCE_DIR}/ir_serialization/models/\"")
```

Comment:
`add_definitions` -> `target_compile_definitions`

Reply:
@jbreazil Could you fix it?

Reply:
Will fix it today.

Reply:
Done in this [commit](https://github.com/openvinotoolkit/openvino/pull/2579/commits/95cc54c5752b78471125fcdb8fcf11b9284d078d)